### ISSUE TITLE
Errors in Regolith with sway 3.2

### ISSUE THREAD

**Issue by rmasad** [regolith-linux/sway-regolith#23]
- Does not allow opening the file selection management window (which prevents attaching to email managers or uploading a file in Chrome)
- Key combinations such as shift+super+n (open Nautilus) or super+w (open Wifi management) do not work.
- Still using sway 1.8 (I was hoping that when I updated, it would update to sway 1.9)

**Comment by SoumyaRanjanPatnaik** [comment]
Hey @rmasad. 
> Does not allow opening the file selection management window (which prevents attaching to email managers or uploading a file in Chrome)

Can you check if `xdg-desktop-portal-regolith-wayland-config` is installed? If yes, which version? This issue is fixed in v0.3.5.

> Key combinations such as shift+super+n (open Nautilus) or super+w (open Wifi management) do not work.

Do you have the `regolith-sway-control-center-regolith` package installed?

> Still using sway 1.8 (I was hoping that when I updated, it would update to sway 1.9)

Which version of Ubuntu/Debian are you using? Sway v1.9 is only available on `noble` and Debian testing for now.

**Comment by rmasad** [comment]
Fixed issues with sway update and key combinations. You had to install regolith-sway-control-center-regolith and update.

What has not been solved is opening the file selection management window

**Comment by SoumyaRanjanPatnaik** [comment]
Can you check if xdg-desktop-portal-regolith-wayland-config is installed?

**Comment by rmasad** [comment]
It's installed

**Comment by rmasad** [comment]
Some additional information: in Gnome the file selection window works, but in Regolith with sway and with i3wm it does not.

**Comment by SoumyaRanjanPatnaik** [comment]
Hmm. Can you let me know if the following commands fix it?
```
systemctl --user import-environment XDG_CURRENT_DESKTOP
exec hash dbus-update-activation-environment 2>/dev/null && dbus-update-activation-environment --systemd XDG_CURRENT_DESKTOP
systemctl restart --user xdg-desktop-portal.service
```

LMK if the file picker appears for you.

**Comment by rmasad** [comment]
Yes, it solves it. Do I keep the issue open for a solution for other users or close it?

**Comment by SoumyaRanjanPatnaik** [comment]
This wil be fixed in Regolith 3.2 beta 2. We can close it. For the meantime, you may add this to your sway configurations.
